rabbit_mq/variables.tf

```diff
@@ -0,0 +1,109 @@
+variable "region" {
+  description = "AWS Default Region"
+  default     = "us-east-1"
+}
+
+variable "profile" {
+  type        = string
+  default     = "default"
+  description = "Credentials profile for AWS provider"
+}
+
+variable "user_tags" {
+  description = "Mandatory tags for the RabbitMQ resources"
+  type        = map(string)
```

Comment:
You can make this as an object to make certain tags required

Reply:
@salyateem13 we are going to leave it as such since it is handled in the tags module.